lib/plugins/package/lib/zipService.js

```diff
@@ -37,17 +33,18 @@ module.exports = {
     if (excludeDevDependencies) {
       this.serverless.cli.log('Excluding development dependencies...');
 
-      return BbPromise.bind(this)
-        .then(() => excludeNodeDevDependenciesMemoized(servicePath))
-        .then(exAndInNode => {
-          params.exclude = _.union(params.exclude, exAndInNode.exclude); //eslint-disable-line
-          params.include = _.union(params.include, exAndInNode.include); //eslint-disable-line
-          return params;
-        })
-        .catch(() => params);
+      let exAndInNode;
+      try {
+        exAndInNode = await excludeNodeDevDependenciesMemoized(servicePath);
+      } catch {} //eslint-disable-line
```

Comment:
Let's not add eslint disablers (why it's added here?)

Reply:
I put this try catch in here to replicate the previous behaviour with `.catch()`. I hadn't looked into what the `excludeNodeDevDependenciesMemoized` call was doing.
Looking now, it appears that `excludeNodeDevDependencies` already has a try / catch fallback so we don't need the try catch at this level. I'll remove it.

Reply:
My comment was strictly about `// eslint-disable-line`, still if `try/catch` doesn't seem justified indeed it's better to remove it